server/server.go

```diff
@@ -382,19 +380,19 @@ func (checker cwfChecker_t) ServeHTTP(writer http.ResponseWriter, req *http.Requ
 		return
 	}
 
-	// Check for user nonce.
-	cliUserName := req.Header.Get("Cwf-User-Name")
-	user, ok := ServerUsers[cliUserName]
+	// Check for user ID.
+	cliName := req.Header.Get("Cwf-User-Name")
```

Comment:
I think this name is misleading. CLI is something else

Reply:
ooouh yes - should be something more accurate